src/components/Points/CampaignPromo.tsx

```diff
@@ -45,7 +50,7 @@ export const CampaignPromo = () => {
       <LinearProgress
         className={css.progressBar}
         variant="determinate"
-        value={progress}
+        value={progress * 100}
```

Comment:
Did something change about the progress value so that we have to multiply it now?

Reply:
It was wrong before but noone realized it. It has to be between 0 and 100 and not 0 and 1